Gemfile.lock

```diff
@@ -272,4 +272,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.3.4
+   2.3.24
```

Comment:
Should this have changed?

Reply:
Hmm I left it because I don't think it's an issue. Maybe I should have gone with latest though, it's a bit random. I can remove it as well.